tools/dockerfile/distribtest/php_jessie_x64/Dockerfile

```diff
@@ -14,4 +14,8 @@
 
 FROM debian:jessie
 
-RUN apt-get update && apt-get install -y php5 php5-dev php-pear phpunit
+RUN apt-get update && apt-get install -y php5 php5-dev php-pear
+
+RUN wget https://phar.phpunit.de/phpunit-5.7.27.phar && \
```

Comment:
There is a problem with this specific file: `wget` was not previously installed.